src/API/Trace/SpanContextValidator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Trace;
+
+use function strlen;
+use function strtolower;
+
+class SpanContextValidator
+{
+    public const INVALID_TRACE = '00000000000000000000000000000000';
+    public const INVALID_SPAN = '0000000000000000';
+    public const SPAN_LENGTH = 16;
+    public const TRACE_FLAG_LENGTH = 2;
+    public const TRACE_LENGTH = 32;
+    public const TRACE_VERSION_REGEX = '/^(?!ff)[\da-f]{2}$/';
+
+    /**
+     * @param string $traceVersion
+     * @return bool Returns a value that indicates whether a trace version is valid.
+     */
+    public static function isValidTraceVersion(string $traceVersion): bool
```

Comment:
`::isValidTraceVersion()` and `::isValidTraceFlag()` are directly related to the W3C propagator - it might be better to move them out of the API into the propagator, especially because `::isValidTraceFlag()` accepts different values than the `$traceFlags` parameter of `::create()`.

Reply:
Created TracePropagationValidator class consisting of above methods..